services/algolia-sync/src/components/algolia/message-sync.js

```diff
@@ -0,0 +1,43 @@
+const algoliasearch = require('algoliasearch');
+const gql = require('graphql-tag');
+const apollo = require('../graphql/query');
+const { buildSections, boostResult } = require('./helpers');
+require('dotenv').config();
+
+const client = algoliasearch(process.env.ALGOLIA_APPID, process.env.ALGOLIA_API_KEY);
```

Comment:
```suggestion
const { ALGOLIA_APPID, ALGOLIA_API_KEY } = require('./env');

const client = algoliasearch(ALGOLIA_APPID, ALGOLIA_API_KEY);
```
Update to use `envalid` syntax.